enclave/core/sgx/backtrace.c

```diff
@@ -112,7 +112,7 @@ char** oe_backtrace_symbols_impl(
         goto done;
 
     /* First call might return OE_BUFFER_TOO_SMALL. */
-    if (oe_backtrace_symbols_ocall(
+    if (oe_sgx_backtrace_symbols_ocall(
```

Comment:
Should also add `_sgx_` into lines 93, 184, and 191.  Similarly in other .c files in this directory.  Do you want to do that in this PR or in a separate PR?

Reply:
The goal of this PR was just to fix up the EDL files before I publish them. Still tracking the need to fix up naming across the board with https://github.com/openenclave/openenclave/issues/2796